app/views/katello/api/v2/organizations/show.json.rabl

```diff
@@ -2,8 +2,7 @@ object @taxonomy
 
 extends "api/v2/taxonomies/show"
 
-attributes :task_id, :label, :system_info_keys, :distributor_info_keys, :default_info,
-  :owner_details, :redhat_repository_url
+attributes :task_id, :label, :owner_info, :redhat_repository_url
```

Comment:
I would note that the above would probably be better suited to a manifest API (which I looked into for re-doing the repository enablement page) but not something I encourage tackling in this PR :)